core-bundle/src/Resources/contao/models/FormModel.php

```diff
@@ -32,6 +32,7 @@
  * @property string  $attributes
  * @property string  $formID
  * @property boolean $allowTags
+ * @property string  $mailer_transport
```

Comment:
Please also adjust the `find*()` and `count*()` methods.

Reply:
Done.